src/messages/pbrt.proto

```diff
@@ -287,115 +287,123 @@ message QueueStats {
     uint64 outstanding_udp = 7;
 }
 
-message WorkerStats {
-  message ObjectRayStats {
+message ObjectRayStats {
     ObjectKey id = 1;
     RayStats stats = 2;
-  }
+}
 
-  message Interval {
+message Interval {
     uint64 start = 1;
     uint64 end = 2;
-  }
+}
 
-  message ActionIntervals {
+message Action {
+    string name = 1;
+    uint64 time = 2;
+}
+
+message ActionIntervals {
     string name = 1;
     repeated Interval intervals = 2;
-  }
+}
 
-  message MetricPoint {
+message MetricPoint {
     uint64 time = 1;
     float value = 2;
-  }
+}
 
-  message Metrics {
+message Metrics {
     string name = 1;
     repeated MetricPoint points = 2;
-  }
+}
 
-  uint64 finished_paths = 1;
-  RayStats aggregate_stats = 2;
-  QueueStats queue_stats = 3;
-  repeated ObjectRayStats object_stats = 4;
-  uint64 cpu_millis = 5;
-  uint64 bytes_sent = 6;
-  uint64 bytes_received = 7;
-  map<string, float> time_per_action = 8;
-  repeated ActionIntervals intervals_per_action = 9;
-  repeated Metrics metrics_over_time = 10;
+message WorkerStats {
+    uint64 finished_paths = 1;
+    RayStats aggregate_stats = 2;
+    QueueStats queue_stats = 3;
+    repeated ObjectRayStats object_stats = 4;
+}
+
+message WorkerDiagnostics {
+    uint64 bytes_sent = 1;
+    uint64 bytes_received = 2;
+    uint64 outstanding_udp = 3;
+    repeated Action time_per_action = 4;
+    repeated ActionIntervals intervals_per_action = 5;
+    repeated Metrics metrics_over_time = 6;
 }
 
 // For stat output to logs.
 message TreeletStatsTrace {
```

Comment:
I believe that this message, and those below, are all dead now. Can we strip them out?